zserio-version.txt

```diff
@@ -0,0 +1 @@
+2.0.0-pre3
```

Comment:
Consider removing this one to avoid redundancy - Is this needed although zserio.jar can be used to query the version?